frontend/src/views/ImageDetailsPage/components/FeatureTiles/TextEntityTile/TextEntityTile.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { FeatureTemplate, FeatureItem } from 'assets/styles/style';
+
+export const TextEntityTile = ({ entity }) => {
+  return (
+    <FeatureTemplate>
+      <p>Text Entity Details</p>
+      <ul>
+        {entity?.map((obj, i) => (
+          <FeatureItem key={i}>
+            <p>{obj.name}</p>
+            <p>type: {obj.type}</p>
+            <p>sentiment: {obj.sentiment}</p>
```

Comment:
```suggestion
            <p>Type: {obj.type}</p>
            <p>Sentiment: {obj.sentiment}</p>
```
Just for the consistency.